src/lobby/lobby.js

```diff
@@ -66,7 +66,7 @@ const TagTableList = tablesList.map((table, index) => (
   <tr key={index}>
     <td>{table.name}</td>
     <td>{table.createdBy}</td>
-    <td>{table.players_num}/{table.max_players_num}</td>
+    <td>{table.playersOnTable.length}/{table.max_players_num}</td>
```

Comment:
as I said, no need for players_num because its the same as the length of the array of the players that on that table